kubernetes-filebeat/templates/daemonset-nodes.yaml

```diff
@@ -35,10 +35,11 @@ spec:
           runAsUser: 0
         resources:
           limits:
-            memory: 200Mi
+            cpu: 1000m
+            memory: 1024Mi
           requests:
-            cpu: 100m
-            memory: 100Mi
+            cpu: 1000m
```

Comment:
1 dedicated core for filebeat ? That's 12.5% of node cpu.
Plus other small utils, we might end with 20% cpu "wasted" to system. This would be sad.

Reply:
Yes, I do not intend to keep that.
Actually It would be good to have resources specified at implementation but I don't understand how can I pass them. Any hint?

Reply:
I have put 500m/500Mi
The volume of logs is not good enough to give good estimates, imo.
With the resources allocated before there were some OOM restarts. Now they should be ok...
Hoping to reduce the requests when we have a stable system again.
Help on passing resources from values is veeery welcome :)